src/api/app/controllers/webui/obs_factory/staging_projects_controller.rb

```diff
@@ -15,8 +15,8 @@ def index
       respond_to do |format|
         format.html do
           @staging_projects = ::ObsFactory::StagingProjectPresenter.sort(@distribution.staging_projects_all)
-          @backlog_requests = Request.with_open_reviews_for(by_group: @distribution.staging_manager, target_project: @distribution.name)
-          @requests_state_new = Request.in_state_new(by_group: @distribution.staging_manager, target_project: @distribution.name)
+          @backlog_requests = ::ObsFactory::Request.with_open_reviews_for(by_group: @distribution.staging_manager, target_project: @distribution.name)
```

Comment:
This changes could be in a8158a0418f208a90227ebe41bdfa7e2f51024ac instead of in its own commit